selendroid-standalone/src/main/java/io/selendroid/standalone/server/handler/NetworkConnectionHandler.java

```diff
@@ -78,9 +78,14 @@ public Response handleRequest(HttpRequest request, JSONObject payload) throws JS
 
 
     // nice and magical, this will 'close' the settings view and reopen the last application, the one under test! :)
-    device.inputKeyevent(KeyEvent.KEYCODE_DPAD_UP);
-    device.inputKeyevent(KeyEvent.KEYCODE_DPAD_CENTER);
-
+    if (deviceAPILevel == 21) {
```

Comment:
should this be `>= 21` ?

also, why are you sending *three* back keys?  Is there something special about 3?  should this be a loop looking for some specific behavior?  (assuming 3 back keys is right, please add a comment.)